azavea/echo-locator
# Edit voucher copy and default selection

## Overview

This PR updates the default "Do you have a voucher?" field to “yes” on the frontend (`True` in admin) as well as updates any user-facing copy that includes “voucher” to specify “Boston Housing Authority voucher” instead. 

### Checklist


### Notes

- I decided to leave any translations that include "voucher" alone. I _could_ pop in the "BHA" label but I think I better keep my hands off until someone can formally translate.
- There's some copy that reference "Client ID" that I think was used for the search page, but is no longer used elsewhere in the app (`Accounts.InvalidVoucherNumber`, `Accounts.MissingVoucherNumber`, `Accounts.Voucher`). I switched it to "Boston Housing Authority voucher" to it's consistent but maybe worth just deleting these lines altogether?

## Testing Instructions

 * `scripts/update`
 * `scripts/server`
 * Navigate to [frontend site](http://localhost:9966/) and create a new user. Confirm "Do you have a voucher?" field defaults to “yes”.
 * Work way through the site to confirm any copy that included "voucher" now specifies "Boston Housing Authority voucher". From what I can find this is mostly limited to the edit profile page for new and verified users. 
 * Navigate to [admin](http://localhost:9966/) and create a new `UserProfile`. Confirm the "Has a Boston Housing Authority voucher" field defaults to "checked".
 * Confirm any other fields on the admin that include "voucher" are now "Boston Housing Authority voucher".


Resolves #543 #544
